File: Selenium/Activities/Activity6.py
from selenium import webdriver
from selenium.webbrowser.common.by import By

with webbrowser.Firefox() as browser:
    browser.get("https://training-support.net/webelements/dynamic-controls");
    print(f"Current page title: {browser.title}")
			
    checkbox = browser.find_element(By.XPATH,"//input[@id = 'checkbox']")
    print(f"Checkbox currently visible: {checkbox.is_displayed()}")
    print(f"Checkbox selected state: {checkbox.is_selected()}")
    checkbox.click()
    print(f"Checkbox selected state: {checkbox.is_selected()}")
    browser.quit()

File: Selenium/Activities/Activity6.java
package examples;

import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;

public class Activity6 {

	public static void main(String[] args) {

			WebDriver browser = new FirefoxDriver();
			browser.get("https://training-support.net/webelements/dynamic-controls");
			System.out.println("Current page title: "+ browser.getTitle());
			
			WebElement checkbox = browser.findElement(By.xpath("//input[@id = 'checkbox']"));
			System.out.println("Checkbox currently visible: " + checkbox.isDisplayed());
			
			System.out.println("Checkbox selected state: " +checkbox.isSelected());
			
			checkbox.click();
				
			System.out.println("Checkbox selected state: " +checkbox.isSelected());
			
			browser.quit();

	}

}
